bin/skel.py

```diff
@@ -21,6 +21,7 @@ def validate(self, document):
 
 except Exception:
     has_questionary = False
+has_questionary = False
```

Comment:
Are you sure you want to always set `has_questionary` to `False`, regardless of whether the import succeeds? :stuck_out_tongue: